### ISSUE TITLE
THE LAST REMNANT Remastered

### ISSUE THREAD

**Issue by GoffyDude** [Ryujinx/Ryujinx-Games-List#1562]
#### Current on `master` : 1.0.4665

Crashes on launch.

#### Log file :

[TLRR.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4711510/TLRR.zip)

**Comment by EmulationFanatic** [comment]
Updated

**Comment by Eleazar8612** [comment]
Hello, good morning, I have the game the last remnant remastered and it runs perfectly on yuzu but it gives me image errors in the battles like colored pictures but it runs at 60 fps and in Ryujinx the images look good but the images are very low fps
What I can do
I need your help
Thank you

**Comment by EmulationFanatic** [comment]
> Hello, good morning, I have the game the last remnant remastered and it runs perfectly on yuzu but it gives me image errors in the battles like colored pictures but it runs at 60 fps and in Ryujinx the images look good but the images are very low fps What I can do I need your help Thank you

Come to Discord for support. This is a compatibility list.

**Comment by Eleazar8612** [comment]
This TLRR.zip file is to be placed somewhere in the emulator

**Comment by EmulationFanatic** [comment]
> This TLRR.zip file is to be placed somewhere in the emulator

No, it's a log file from the test performed. Please come to Discord for support if you require it.

**Comment by pinguallyj** [comment]
Tested on MacOS M2, vulkan:
Game Update Version : 1.0.1
Ryujinx version: 1.1.0
Game has random stutters and dips to 45 FPS very often. Sometimes the textures start flickering and UI in battle too. Battle loading is quite long around 7-9 seconds while on the console it takes 2-3 seconds (even with the enemies you have fought before, so the shaders are not loading)
